doc/features.mld

```diff
@@ -350,7 +350,7 @@ have no destination. This example is rendered {{!Odoc_examples.Expansion.TypeSub
 The OCaml construct [module type of] allows the type of a module to be
 recovered. As usual, when OCaml performs this operation, it only retains
 the simplified signature, stripped of comments, includes, and more
-complex module type expressions. As with the previous sections, [odoc]
+complex module-type expressions. As with the previous sections, [odoc]
```

Comment:
It is "module type" expression and not module "type expression" but I still thinks that it should not have a dash as per https://ocaml.org/manual/5.2/modtypes.html !